### DISCUSSION TITLE
Implement custom TypeDoc config file

### DISCUSSION THREAD

**Discussion by ryceg** [skeletonlabs/skeleton#506] [Ideas]
### Describe what feature you'd like. Pseudo-code, mockups, or screenshots of similar solutions are encouraged!

Implementing a config for TypeDoc will enforce styling on the custom tags that will be developed; if we stick a `@required` tag in there and treat it incorrectly as a block tag, it'll be able to yell at us.

### What type of pull request would this be?

Enhancement

### Any links to similar examples or other references we should review?

https://typedoc.org/guides/options/#configuration-options

**Comment by ryceg** [comment]
This is of course with the caveat that we opt for TypeDoc, which would only have a marginal benefit considering that our docs are generated via svelte- type "safety" for tags would be the main benefit of that dev dependency.

**Comment by endigo9740** [comment]
I like the concept!

**Comment by endigo9740** [comment]
@ryceg Let's keep this idea in mind when custom tags are a thing. Don't want to put the cart before the horse here.

**Comment by endigo9740** [comment]
Just noticed I have a "convert to discussion" option so I'm going to do that!
